azure-stack/hci/manage/manage-secure-baseline.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Manage baseline security settings on Azure Stack HCI, version 23H2 (preview)
+description: Learn about managing baseline security settings available for new deployments of Azure Stack HCI, version 23H2 (preview).
+author: alkohli
+ms.author: alkohli
+ms.topic: how-to
+ms.service: azure-stack
+ms.subservice: azure-stack-hci
+ms.date: 12/04/2023
+---
+
+# Manage baseline security settings for Azure Stack HCI, version 23H2 (preview)
+
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article describes how to manage baseline security settings for your Azure Stack HCI cluster and the associated drift control mechanism to ensure that the device starts in a known good state.
+
+[!INCLUDE [important](../../includes/hci-preview.md)]
+
+## Enable, disable, or toggle drift control
+
+To adjust security hardening, we recommend a balanced security posture. Use the initial security baseline, toggle drift control, and modify protected security settings defined during deployment.
+
```

Comment:
Start with the initial security baseline and then modify the drift control and protected security settings defined during the deployment.

Reply:
Done > updated.